schemas/iso19139/src/main/plugin/iso19139/schema.xsd

```diff
@@ -29,7 +29,12 @@
            xmlns:gmx="http://www.isotc211.org/2005/gmx"
            elementFormDefault="qualified"
 >
+  <!-- 19139:2007 version -->
+  <xs:include schemaLocation="schema2007/gmd/metadataApplication.xsd"/>
+  <xs:import namespace="http://www.isotc211.org/2005/srv" schemaLocation="schema2007/srv/srv.xsd"/>
+  <xs:import namespace="http://www.isotc211.org/2005/gmx" schemaLocation="schema2007/gmx/gmx.xsd"/>
+  <!-- 19139:2005 version
```

Comment:
Should be this enabled manually for 2005 version? How affects the validation if the system has metadata in both versions?

Reply:
> Should be this enabled manually for 2005 version?

Yes.



> How affects the validation if the system has metadata in both versions?

This is a situation to avoid. At least you can't validate both schema unless you create a dedicated plugin for each.

Reply:
Yep seems the idea is to move everything to the new schemas and not have this situation. Seems good idea to me as I think it is too messy otherwise and no reason not to either.